exonum-java-binding/launcher-plugins/README.md

```diff
@@ -0,0 +1,138 @@
+# Java Runtime Plugins for [Exonum Launcher](https://github.com/exonum/exonum-launcher)
+
+- Java Runtime Plugin allows to deploy and start Java services.
+- Java Instance Plugin allows to use an arbitrary Protobuf message for service 
+  initial configuration parameters. The plugin can be used for both Java and Rust services.
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+
+```bash
+# Generate sources
+mvn generate-sources -pl core
+# Install plugins
+python3 -m pip install -e launcher-plugins
+```
+
+## Usage
+
+TODO: Move the entire section to the EJB App tutorial.
+
+To deploy and start a specific list of services, use the following command with a
+prepared `config.yml` file:
+
+```bash
+python3 -m exonum_launcher -i config.yml
+```
+
+### Writing Configuration File
+
+Start with specifying IP addresses of the blockchain nodes:
+
+```yaml
+networks:
+  - host: "127.0.0.1"
+    ssl: false
+    public-api-port: 8080
+    private-api-port: 8081
+```
+
+You need to specify every node in the network.
```

Comment:
Does it require the private ports of nodes to be accessible from a single location? I wonder if there are any other operation on the private port that do not authorize the user, making such setup impractical in distributed scenarious. Is that a known, deliberate restriction?

Reply:
ping @popzxc

Reply:
`exonum-launcher` provides functionality to interact with nodes that you own. If you own all the nodes, you can send all the requests from one launch.
If network has several administrators, every administrator has to run `exonum-launcher` with the networks he owns.

Decentralized supervisor (as the oppose to the simple supervisor, which requires 1 request only, and which is now default) requires N requests and `exonum-launcher` is *trying* to make the life of the developer better, but this is not a generic solution for every situation.

Reply:
Thanks for clarifications!

@vitvakatu , The documentation seems incomplete and has to be updated then, doesn't it?

Reply:
@dmitry-timofeev it seems, I will update.